change/@internal-react-components-3b1b621a-3d43-4490-9374-b067034d374d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
There should be a changelog for calling-component-bindings as well

Reply:
I thought so as well but the tool didn't ask for one?

Reply:
Stacked PR (where the base PR also has one). The rush changelog compares against main and not an arbitrary base branch, so once the base PR goes in it will complain you don't have one